src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -82,73 +75,68 @@ const defaultProps = {
     backButtonRoute: '',
 };
 
-class WorkspacePageWithSections extends React.Component {
-    componentDidMount() {
-        this.fetchData();
-    }
-
-    componentDidUpdate(prevProps) {
-        if (!prevProps.network.isOffline || this.props.network.isOffline) {
-            return;
-        }
-
-        this.fetchData();
+function fetchData(skipVBBACal) {
+    if (skipVBBACal) {
+        return;
     }
 
-    fetchData() {
-        if (this.props.shouldSkipVBBACall) {
-            return;
-        }
-
-        BankAccounts.openWorkspaceView();
-    }
+    BankAccounts.openWorkspaceView();
+}
 
-    render() {
-        const achState = lodashGet(this.props.reimbursementAccount, 'achData.state', '');
-        const hasVBA = achState === BankAccount.STATE.OPEN;
-        const isUsingECard = lodashGet(this.props.user, 'isUsingExpensifyCard', false);
-        const policyID = lodashGet(this.props.route, 'params.policyID');
-        const policyName = lodashGet(this.props.policy, 'name');
-
-        return (
-            <ScreenWrapper
-                includeSafeAreaPaddingBottom={false}
-                shouldEnablePickerAvoiding={false}
+function WorkspacePageWithSections(props) {
+    const {children, footer, shouldSkipVBBACall} = props;
```

Comment:
Hello, @arosiclair 
Nice to meet you.
It is possible to use the destructured props params?
I thought it prohibit to use destructured props params because all of function components are using props params directly.

Reply:
@arosiclair 
I just destructured props.
Please check again.
Thanks